onepager.py

```diff
@@ -300,9 +308,9 @@ def say_hi(self):
 # Data Classes
 @dataclass
 class DataClass:
-    member1: int
-    member2: float
-    member3: str
+    member1: int                                # Typehints are mandatory for data classes but not enforced at runtime
+    member2: Any                                #  since Python is dynamically typed. Use Any if you don't want to
```

Comment:
**note**: Note against including how to use `Any` but the Python community is trying to climb out of a hole of being dynamically typed and `DataClass` was invented to have a very specific, seemingly typed object. If you use something like `MyPy` it will check that you're using the correct types in the codebase. `Any` is really just something for backwards compatibility if you can't control the type. For most dynamic purposes, you actually use `Union` and be able to specify the specific types you might use. For example, `Union[int, str]`.

Reply:
Okay, reworded the comment some. Maybe we can add an example with type-checking. I don't have any experience with it so I'll read a type-checking guide tonight.